### ISSUE TITLE
Possible to sync gnome-shell top panel bar with upstream pixel size?

### ISSUE THREAD

**Issue by Scotty-Trees** [ubuntu/yaru#3384]
Greetings again,

**Expected Behavior** 

I don't know if my report is a bug issue per se or more a purposeful choice by the Yaru team, but I'll do my best to offer as much information as I can in the matter. One thing I've noticed recently is that the Yaru theme is doing their best to align more and more with the upstream Adwaita theme and I like that approach. One thing I have noticed that isn't or hasn't yet been synced with upstream is the gnome-shell top panel bar is fairly smaller in pixel size when compared to the gnome-shell top panel bar of the Adwaita theme. I'll post a few screenshots and a recording below to clarify.

One thing I would like to know if you'll humor me is if this could be changed to be in sync with upstreams top panel size or if the smaller top panel bar is a intentional design choice by the Yaru team; if you could specify that I'd appreciate it very much. 

When you use Gnome Tweaks, to switch between Adwaita and Yaru, you'll notice the top panel will shrink a little bit with the latter. Since Gnome is going for a slightly larger and wider look across their entire theme (and so have a lot of Yaru elements as well), I think it would be good to sync this with upstream as well as it makes the top panel just a bit more easier to read, but also easier to click and navigate around. It is a small change don't get me wrong, but I think being as close to upstream as possible is a good approach and this is just one area I noticed could you some attention. I think the Yaru team is doing a wonderful job lately, and all the recent commit updates are a welcome for sure, so let me just say thank you again for all your work and dedication. 

**System information**

- Shell version: 
```
[scott@endeavourOS ~]$ gnome-shell --version
GNOME Shell 41.3
```

- OS version: 
```
[scott@endeavourOS ~]$ lsb_release -a
LSB Version:	1.4
Distributor ID:	EndeavourOS
Description:	EndeavourOS Linux
Release:	rolling
Codename:	rolling
```

**Yaru version**

```
aur/yaru-gnome-shell-theme-git 1:21.10.1.r50.g670a52f26-1 (+4 0.04) (Installed: 1:21.10.1.r332.ged2d6ee74-1)
```

I use the AUR version of Yaru, which is essentially Yaru git built from source from the latest commit. Here's the AUR link if that's helpful:

https://aur.archlinux.org/packages/yaru-gnome-shell-theme-git/

**Screenshots/Video**

![Screenshot from 2022-02-10 23-23-46](https://user-images.githubusercontent.com/45135187/153538996-56c16910-5b67-4899-b1d3-5bac97080f0a.png)
![Screenshot from 2022-02-10 23-24-14](https://user-images.githubusercontent.com/45135187/153539003-3a64af2d-1841-4205-823c-bd69d1235f9d.png)

https://user-images.githubusercontent.com/45135187/153539264-89feb1bd-745f-4ac1-a08f-e9fbb2a625e9.mp4

As a quick side note, I've tested GnomeOS Nightly (showcases the latest Gnome updates) in a Boxes VM and the top panel bar is still the same size in Gnome 42 as it is currently in Gnome 41 as far as my eyes can tell. Would love to see possibly synced up with upstream, I personally love the Adwaita theme, but I like the darker colors that Yaru provides.

 If you need any additional information, just let me know an I hope this helps any and thanks again for making an amazing theme, you all are the best. 

tl;dr, could it be possible to increase the pixel size of the gnome-shell top panel bar to be in sync with upstream

**Comment by Scotty-Trees** [comment]
It's been 2 weeks since the post. Any thoughts? Apologizes I do not mean to pester, I very much enjoy this project and the work everyone involved does. I just hope the shell can be more like upstream.

**Comment by Muqtxdir** [comment]
Hi @Scotty-Trees 

Thanks for the appreciation and kind words towards yaru and for taking out time for opening this issue.

sorry, for the late-reply but this top-bar issue will be discussed/addressed along with the GS42 refinements issue: #3407

**Comment by Jupi007** [comment]
I'm really sorry @Scotty-Trees.
I'm really busy these day and didn't found the time to respond you.

Your issue is very well written with good arguments.
We'll think about it :)

**Comment by Scotty-Trees** [comment]
@Muqtxdir no worries about the late reply, I understand everyone can be busy with things. Thanks for the #3407 tracker, I'll be sure to follow and keep up with that for any updates.

@Jupi007 it's totally fine, I know you all are busy creating a great theme, and it's okay if sometimes things or life gets in the way it happens; it always helps to be flexible and open minded to others, so don't worry about it you're all good.

As a quick side note, I'm sure this is just a temporary glitch, but I just updated to the latest commit [1:21.10.1.r362.ga4c4f2815-1] (logged out and back in) as of writing this and my volume up/down notification looks very large and in charge now. I've seen the Gnome 42 notification mock ups and I think this may need a tweak ;) (if it's hard to tell I am only be a bit sarcastic and not trying to be crude in any way):

![Screenshot from 2022-02-24 15-28-17](https://user-images.githubusercontent.com/45135187/155603321-38174d2e-a8ac-4bf3-b135-d65d2796d872.png)

**Comment by Muqtxdir** [comment]
> As a quick side note, I'm sure this is just a temporary glitch, but I just updated to the latest commit [1:21.10.1.r362.ga4c4f2815-1] (logged out and back in) as of writing this and my volume up/down notification looks very large and in charge now. I've seen the Gnome 42 notification mock ups and I think this may need a tweak ;) (if it's hard to tell I am only be a bit sarcastic and not trying to be crude in any way):

_this is happening because you are  on GS42, see below it displays it correctly_ 

![Screenshot from 2022-02-25 14-51-08](https://user-images.githubusercontent.com/54065734/155689610-ad75bd4c-dc05-4533-be0a-1f5ff186f992.png)

**Comment by Scotty-Trees** [comment]
@Muqtxdir I'm sorry if I wasn't totally clear before, but that screenshot with the volume notification being a giant circle is on my Gnome 41.4 install using the latest git commit of the yaru shell and yaru gtk theme. I assume all the upstream changes being made will make yaru look great on Gnome 42, but I think in that process, one or more of those changes has "broken" this volume notification in Gnome 41. Is anyone else able to replicate this maybe?

**Comment by Muqtxdir** [comment]
edit: it should have been written as _not on GS42_ in my above comment

**Comment by Scotty-Trees** [comment]
@Muqtxdir Ahh okay thank you, that makes more sense now. I'm on EndeavourOS, so typically Arch-based distros don't get the latest Gnome until usually the first point release, so I'll have to wait till 42.1 gets released most likely before I can truly be of any further help I think. I guess it won't be wise for me to keep testing the latest yaru theme updates if I'm using the wrong Gnome version. There's an older, but 'stable' yaru gtk theme in the AUR I can substitute for the time being so it's not a total loss. 

I wish you and all the devs much good luck working on the theme in the coming weeks/months. I guess for now I'll report back when I'm using a version of Gnome that it's meant for ;)

**Comment by Muqtxdir** [comment]
hi, so late on housekeeping but somewhere down the line we moved to upstream pixel size 😅
